public/js/replay.js

```diff
@@ -229,6 +229,19 @@ function Replay (params) {
   * there are no more flags on either side, disable it.
   */
   this.checkFlagButtons = function(){
+
+    if(this.prev_flag == null || this.flagged.length === 0){
+      $('#flag-left').prop("disabled",true);
+    }else{
+      $('#flag-left').prop("disabled",false);
+    }
+    if(this.next_flag == null || this.flagged.length === 0){
```

Comment:
Use `===` to compare with null.

https://travis-ci.org/VisionistInc/jibe/jobs/69897059